src/content/testing/common-errors.md

```diff
@@ -443,27 +443,48 @@ attempting to trigger a `Dialog` from within the
 immediately show information to the user,
 but `setState` should never be called from a `build` method.
 
-The following snippet seems to be a common culprit of this error:
+The following code illustrates a common culprit of this error:
 
 <?code-excerpt "lib/set_state_build.dart (problem)"?>
 ```dart
```

Comment:
If all the code required for running an app is being added here, maybe it could be structured as an interactive DartPad sample! I'm not sure exactly how it works but I did see another page that was formatted this way:

https://github.com/flutter/website/blob/93fe19aca35566afda7bb12b78ff952d401586b8/src/content/cookbook/design/tabs.md?plain=1#L96-L136

Reply:
That's a great idea, but I don't know how I'd be able to test the functionality before submitting it for review. I can get common-errors.md onto my PC, but how would I serve it to myself for testing, especially with the an interactive DartPad in place?